### DISCUSSION TITLE
R Shiny: Deactivating action-button

### DISCUSSION THREAD

**Discussion by naturallyfunny** [teamalgoritma/community#770] [Q&A] [answered]
## Pertanyaan:
 Bagaimana cara "mematikan" action-button yang telah di-click?
> Action-button pada case ini adalah ***input$rank***
 
 ## Server Code:
  ```
output$third_row <- renderUI({
    req(input$rank)
    
    # If rank button is clicked but resumes are not submitted
    if (is.null(input$resumes)) {

      >> Saya ingin "mematikan" button input$rank di sini <<

      # Show warning modal
      showModal( 
        modalDialog(
          title = "⚠️No Resumes/CVs Submitted⚠️",
          "Where are the resumes? 😦",
          easyClose = TRUE,
          footer = NULL
        )
      )
    }
    else{
      ~the rest of the code~
    }
```

**Comment by divakartika** [comment]
Dear @ArdiantheGreat 

Dapat dijelaskan tujuan "mematikan" action button ini untuk apa, atau seperti apa output yang diharapkan? Dari [shinyapp yang telah terbentuk](https://ardian.shinyapps.io/Resume_Ranking-Tool/) sepertinya tidak ada masalah yang signifikan.

**Reply by ramdhinafinita** [reply, marked as answer]
Dear @ArdiantheGreat 
Jika masih terkendala error, berikut ini mungkin bisa membantu menjawab pertanyaan Bapak. Bapak bisa coba menggunakan `shinyjs::disable("rank")` dan memasukkannya sebelum `# Show warning modal`. Sehingga bapak dapat menonaktifkan button input dengan ID "rank". **Jika kondisi yang dimasukkan memenuhi**, **button** tersebut akan menjadi tidak dapat diklik **(disabled)**.

referensi: https://stackoverflow.com/questions/66188379/shiny-how-to-start-application-with-action-button-disabled
semoga membantu 🙂

**Comment by ramdhinafinita** [comment]
Halo @ArdiantheGreat, untuk menjaga agar forum diskusi berjalan kondusif, kami izin melakukan close pada diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami.
